src/algonauts/models/model_loaders.py

```diff
@@ -9,5 +10,5 @@ def load_vgg16():
 
 
 def load_from_file(model_filename, transform_image):
-    model = tf.keras.models.load_model(model_filename)
+    model = tf.keras.models.load_model(model_filename, custin_objects={'F1Score': tfa.metrics.F1Score})
```

Comment:
should be "custom_objects" I guess ? :)

Reply:
this is weird, i tested and it didn't complain like without passing it, fixed now